dsc_lib/src/dscresources/command_resource.rs

```diff
@@ -815,26 +816,94 @@ pub fn log_stderr_line<'a>(process_id: &u32, trace_line: &'a str) -> &'a str
 {
     if !trace_line.is_empty()
     {
-        if let Result::Ok(json_obj) = serde_json::from_str::<Value>(trace_line) {
+        if let Ok(trace_object) = serde_json::from_str::<Trace>(trace_line) {
+            let mut include_target = trace_object.level == TraceLevel::Debug || trace_object.level == TraceLevel::Trace;
+            let target = if let Some(t) = trace_object.target.as_deref() { t } else {
+                include_target = false;
+                ""
+            };
+            let line_number = if let Some(l) = trace_object.line_number { l } else {
+                include_target = false;
+                0
+            };
+            let trace_message = if include_target {
+                format!("Process {process_id}: {target}: {line_number}: {}", trace_object.fields.message)
```

Comment:
I feel like we should get rid of the `Process ` word in all traces - it takes too much real estate in the output.
Maybe just have `process_id` in square brackets instead.

Reply:
I'll shorten it to `pid`